Meta/CMake/libtextcodec_generators.cmake

```diff
@@ -11,5 +11,7 @@ function (generate_encoding_indexes)
         arguments -j "${LIBTEXTCODEC_INPUT_FOLDER}/indexes.json"
     )
 
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/LookupTables.h" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/LibTextCodec/")
+    if (ENABLE_INSTALL_HEADERS)
```

Comment:
We should be able to do this by attaching the files to a specific target and setting it to the development component Then the installer could just try to install the runtime components and not get the include files. But I don't think we're 100% setup for that.

Reply:
I tried that, but unfortunately CMake doesn't have an "install everything except component X" `install` invocation. Therefore, everything that involves that would start to bitrot.

Reply:
Meh, yeah. Needs a bit of a redesign probably. Not sure what other applications do as far as headers go. Likely we can get away with not installing headers 99% of the time.